dining/constants.py

```diff
@@ -0,0 +1,104 @@
+pretty_venue_names = {
```

Comment:
A comment right above this explaining why these names are useful would be nice :)